src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -31,6 +31,8 @@ namespace System.Collections
 #if FEATURE_NETCORE
     [FriendAccessAllowed]
 #endif
+    /// <summary>Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is dynamically increased as required.To browse the .NET Framework source code for this type, see the Reference Source.</summary>
```

Comment:
> Can we change the tool to have a max length on the content, such that beyond that it uses multiple lines and wraps

@Priya91, can you comment here?  We use the Roslyn code formatter, so if we can configure it to do this, then yes.

> Also, why do we need that last sentence about browsing the Reference Source?

This is in the intellisense files, so we'll have to remove it either in those files, or during the PR process on a per-file basis.

>And, what about the Remarks section of the docs? Should that be brought in as remarks here? And what's the plan for other aspects of the docs, e.g. the "Thread Safety" section?

@mairaw could clarify here, but I don't believe `<remarks>...</remarks>` comments are generated when building intellisense files.  I can't find any remarks or Thread Safety sections in the intellisense files I've been working with.